x-pack/plugins/observability/public/components/app/section/index.tsx

```diff
@@ -25,36 +25,31 @@ interface Props {
 export function SectionContainer({ title, appLink, children, hasError }: Props) {
   const { core } = usePluginContext();
   return (
-    <EuiAccordion
-      initialIsOpen
-      id={title}
-      buttonContentClassName="accordion-button"
-      buttonContent={
-        <EuiTitle size="s">
-          <h5>{title}</h5>
-        </EuiTitle>
-      }
-      extraAction={
-        appLink?.href && (
-          <EuiLink href={core.http.basePath.prepend(appLink.href)}>
-            <EuiText size="s">{appLink.label}</EuiText>
-          </EuiLink>
-        )
-      }
-    >
-      <>
-        <EuiSpacer size="s" />
-        <EuiPanel hasShadow>
-          {hasError ? (
-            <ErrorPanel />
-          ) : (
-            <>
-              <EuiSpacer size="s" />
-              {children}
-            </>
-          )}
-        </EuiPanel>
-      </>
-    </EuiAccordion>
+    <EuiPanel>
+      <EuiAccordion
+        initialIsOpen
+        id={title}
+        buttonContentClassName="accordion-button"
+        buttonContent={
+          <EuiTitle size="s">
+            <h5>{title}</h5>
+          </EuiTitle>
+        }
+        extraAction={
+          appLink?.href && (
+            <EuiLink href={core.http.basePath.prepend(appLink.href)}>
+              <EuiButton size="s">{appLink.label}</EuiButton>
+            </EuiLink>
```

Comment:
Do you still need the `EuiLink` in that case?
```suggestion
  <EuiButton href={core.http.basePath.prepend(appLink.href)}>{appLink.label}</EuiButton>
```

Reply:
@cauemarcondes Good point! Thanks